addons/mail/static/src/core/common/message_model.js

```diff
@@ -534,8 +534,8 @@ export class Message extends Record {
         { mentionedChannels = [], mentionedPartners = [], mentionedRoles = [] } = {}
     ) {
         if (
-            createElementWithContent("div", body).textContent ===
-                createElementWithContent("div", this.body).textContent &&
+            createElementWithContent("div", body).innerHTML ===
+                createElementWithContent("div", this.body).innerHTML &&
```

Comment:
This doesn't always work.

In particular if I do enter and then revert it, it's considered as different.

Reply:
There is sometimes `<span class="o-mail-Message-edited"></span>` in the message which gets in the way.